channels/pjsip/dialplan_functions_doc.xml

```diff
@@ -55,6 +55,43 @@
 		</description>
 	</application>
 
+	<application name="PJSIPNotify" language="en_US">
+		<synopsis>
+			Send a NOTIFY to either an arbitrary URI, or inside a SIP dialog.
+		</synopsis>
+		<syntax>
+			<parameter name="to" required="false">
+				<para>Abritrary URI to which to send the NOTIFY.  If none is specified, send inside
+				the SIP dialog for the current channel.</para>
+			</parameter>
+			<parameter name="headers" required="true">
+				<para>Either a pre-configured option or a list of headers to send in the NOTIFY.</para>
```

Comment:
> a pre-configured option

... in pjsip_notify.conf maybe?

> or a list of headers 

Can you not set the body with the dialplan app?